README.md

```diff
@@ -1,24 +1,19 @@
-OpenMC Plot Explorer
-====================
+# OpenMC Plot Explorer
```

Comment:
Are these header designations generally preferred over `===` or `----`?

Reply:
For markdown, yes. The fact that underlines work is probably just a convenience for people converting from other markup formats (e.g. rst)